server/src/routes/api/v1/moviesRouter.js

```diff
@@ -17,9 +22,15 @@ moviesRouter.get("/:id", async (req, res) => {
     try {
         const movie = await Movie.query().findById(id)
         const reviews = await movie.$relatedQuery("reviews")
-        movie.reviews = reviews
+        const serializedReviews = await Promise.all(reviews.map(async review => {
+           return await ReviewSerializer.showDetails(review)
+        }
+        ))
+        movie.reviews = serializedReviews
+
         return res.status(200).json({ movie: movie })
     } catch (error) {
+        console.log(error)
```

Comment:
remove console logs from final code. this is fine to add while you are developing to help see errors, but should be removed for final code